src/main/java/org/symphonyoss/symphony/messageml/elements/Dialog.java

```diff
@@ -122,16 +122,29 @@ private void checkAttributeOrPutDefaultValue(String attributeName, String defaul
   }
 
   private void validateChildrenTypes() throws InvalidInputException {
-    assertContainsAlwaysChildOfType(Collections.singleton(DialogChild.Title.class));
-    assertContainsAlwaysChildOfType(Collections.singleton(DialogChild.Body.class));
-    validateNoOtherChildrenTypes();
+    if (getChildren().size() > 1 && getChildren().stream().anyMatch(element -> element instanceof Form)){
+      throw new InvalidInputException("A [dialog] element can't contain a [form] element and any other element.");
+    } else if (getChildren().size() == 1 && getChild(0).getClass().equals(Form.class)) {
+      Element formElement = getChild(0);
+      validateChildren(formElement);
+      validateNoOtherChildrenTypes(formElement, "A [form] element in a [dialog] element can only contain tags \"title\", \"body\", \"footer\"");
+    } else{
+      validateChildren(this);
+      validateNoOtherChildrenTypes(this, "A dialog can only contain tags \"title\", \"body\", \"footer\"");
```

Comment:
could we use the same format for this two message errors? Line 130 and line 133? Like wrap the element name with \" ?